drape_frontend/visual_params.cpp

```diff
@@ -217,6 +217,7 @@ int GetTileScaleBase(ScreenBase const & s)
 int GetTileScaleBase(m2::RectD const & r)
 {
   double const sz = std::max(r.SizeX(), r.SizeY());
+  ASSERT_NOT_EQUAL(sz, 0., ("Rect should not be a point:", r));
```

Comment:
nit: IMO the comment in assert is a tautology.
ASSERT_GREATER(sz, 0, ())
is enough

Reply:
Not every reader will understand that it's zero in a point case :)